src/ansys/fluent/core/utils/deprecate_args.py

```diff
@@ -18,14 +27,16 @@ def wrapper(*args, **kwargs):
             """Warns about the deprecated argument and replaces it with the new
             argument."""
             if old_arg in kwargs:
-                warnings.warn(
-                    f"'{old_arg}' is deprecated. Use '{new_arg}' instead.",
-                    deprecation_class,
-                )
-                val = converter(kwargs[old_arg])
-                if val is not None and kwargs.get(new_arg) is None:
-                    kwargs[new_arg] = val
+                old_value = kwargs[old_arg]
+                warning_str = f"'{old_arg}' is deprecated. "
+                new_value = converter(kwargs[old_arg])
+                if kwargs.get(new_arg) is None:
+                    kwargs[new_arg] = new_value
+                    warning_str += f"Use '{new_arg} = {_str_repr(new_value)}' instead of '{old_arg} = {_str_repr(old_value)}'."
+                else:
+                    warning_str += f"Use only '{new_arg}' instead."
                 kwargs.pop(old_arg)
+                warnings.warn(warning_str, deprecation_class, stacklevel=2)
```

Comment:
`stacklevel=2` means that the warning will indicate what call exactly resulted in the deprecation warning, rather than where in the code the deprecation warning is raised

Before:
`C:\ansysdev\pyfluent\src\ansys\fluent\core\utils\deprecate_args.py:39: PyFluentDeprecationWarning: ...`

Now with `stacklevel=2`:
`C:\tests\test_deprecation.py:3: PyFluentDeprecationWarning: ...`
Which indicates exactly the file name and line number of the call that includes the deprecated argument